apps/api-auth/src/modules/auth/auth.module.ts

```diff
@@ -13,6 +14,17 @@ import { AuthService } from './auth.service';
             secret: process.env.JWT_SECRET || 'default_jwt_secret',
             signOptions: { expiresIn: '15m' },
         }),
+        ClientsModule.register([
```

Comment:
Use injection ConfigService wherever possible, and import your own ConfigModule in the Nest module settings.

In this way:
```suggestion
ClientsModule.registerAsync([
  {
    name: "RABBITMQ_CLIENT",
    imports: [ConfigModule],
    inject: [ConfigService],
    useFactory: (configService: ConfigService) => ({
      transport: Transport.RMQ,
      options: {
        urls: [configService.get<string>("RABBITMQ_URL") ?? "amqp://guest:guest@rabbitmq:5672"],
        queue: configService.get<string>("RABBITMQ_QUEUE") ?? "default_queue",
        queueOptions: {
          durable: false,
        },
      },
    }),
  },
]),
```